docs/_posts/2019-03-05-go.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Getting GOing"
+date: 2019-03-05 09:00:00 -0600
+author: Sigal Sax
+categories: blog
+published: true
+image: ship_it.jpg
+thumb: ship_it.jpg
+image-alt: Test post
+excerpt: "For the past year, we on the Cyberark Conjur team have been working on our latest open source project in Go..."
+---
+
+From the very beginning, the original Golang developers had a clear goal - retain the positive attributes of the common programming languages while building one that was readable, simple, and high performing. Go’s light-weight model for fast compilation, extensive toolset, and native capabilities are what have propelled it toward worldwide adoption and success.  With these modern features and being backed by Google, it is no wonder why open source projects such as Docker, Kubernetes, and some in Cyberark, are being built in Go.  
+
+## Get GOing with Secretless  
```

Comment:
Thinking of changing the title of this subsection b/c that's the name of the post

Reply:
I think it's fine

Reply:
👍